### ISSUE TITLE
Filter author list

### ISSUE THREAD

**Issue by dbeucke** [LibreCat/LibreCat#198]
The A-Z filter in author list (http://demo.librecat.org/person) doesn't work.

http://demo.librecat.org/person shows all authors. By klick on any character nothing happens.

Tested on demo server (Einstein and test user have publications).

**Comment by christianmahnke** [comment]
This will be a hard one to fix:
If you look into [`lib/LibreCat/App/Search/Route/person.pm` line 22](https://github.com/LibreCat/LibreCat/blob/master/lib/LibreCat/App/Search/Route/person.pm#L22), you see that the code constructs a CQL query, basicly like `lastname=e` (don't get fooled by the fact that the field is named `last_name` with underscore in our metadata, it is right in this place, but thats another issue).
If one would make the assumption that we can also use CQL on the command line like `bin/librecat user list 'lastname=e*'` then you can see the problem. There is no result at all. 
The Elasticsearch query generated for this is:
```
curl -XGET 'http://localhost:9200/librecat/researcher/_search?from=0&pretty=1&scroll=1m&size=100' -d '
{
   "query" : {
      "query_string" : {
         "query" : "last_name:\"e*\""
      }
   }
}
'
```
Which is probably wrong.
If one tries to do ` bin/librecat user list 'lastname=e\*'` you'll get:
```
curl -XGET 'http://localhost:9200/librecat/researcher/_search?from=0&pretty=1&scroll=1m&size=100' -d '
{
   "query" : {
      "match" : {
         "last_name" : {
            "query" : "e*"
         }
      }
   }
}
'
```
Is even worse since the [match operation](https://www.elastic.co/guide/en/elasticsearch/reference/1.7/query-dsl-match-query.html) doesn't do wildcards.

**Comment by phochste** [comment]
Fixed with the #220 bug
